WPMedia/fusion-news-theme-blocks
# Themes 1191 alert bar hover styles

**This is my first ticket & PR for Arc, so if anything is wrong please let me know**
**This PR goes with an `arc-themes-feature-pack` PR. You can find that [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/418)**

## Description

The goal of this ticket is to update the hover style of the `alert-bar-block` based on the [hover states audit](https://paper.dropbox.com/doc/Themes-hover-states-audit--B400R8DxsjiVy0YImyMBZ0B4Ag-JyHvxHuZttlIfj5BzjTI3). I made the following changes to the `news` and `commerce` JSON files in the `alert-bar-block/themes` directory:
* Set the alert bar, link, and 'X' buttons to have zero drop shadow
    * **Note** that the alert bar in the audit has drop shadows, but my local PageBuilder repo didn't. The manual zero drop shadow is an added layer of redundancy to be safe
* Set the text color of the link to be `#232B39` when it's hovered over
    * The color is based on the `$text-color` in [this design guide](https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=7053-88355&mode=design&t=VyUFCKGTtubHs8Jt-0)
* Updated the news and commerce styling in `.storybook/themes` to reflect the changes made to the block's `themes` directory
* As a result of these changes, some changes were also automatically made to the `site-styles` directory in `arc-themes-feature-pack`
    * You can find a link to that PR [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/418)
    * These are needed to run this code

## Jira Ticket

[THEMES-1191](https://arcpublishing.atlassian.net/browse/THEMES-1191)

## Acceptance Criteria

1. Drop shadow removed
2. Hover: text color changed from white to `$text-color` (`#232B39`)

## Test Steps

1. Checkout this branch `git checkout THEMES-1191-alert-bar-hover-styles`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/alert-bar-block`
3. Open your local PageBuilder instance and open the 'Homepage' and set the Website to 'The Gazette'
    * Any other page with an alert block works too, like 'All Blocks-Test-TA-Sanbox'
    * You can open the preview mode if you want, but the changes can be checked in the editor too
4. Verify that there are no drop shadows and that the link in the alert bar turns `#232B39` when the mouse hovers over it
    * Try it on multiple devices to make sure the default and desktop versions have the same styling

## Effect Of Changes

### Before

* The link and 'X' have drop shadows
* The link turns black when hovered over with the mouse

**Note:** If the GIF isn't playing on Github, you can look at it [here](https://paper.dropbox.com/doc/Themes-hover-states-audit--B6sjeNtMFcuEmlvZBfVsn4RIAg-JyHvxHuZttlIfj5BzjTI3). It's the one under the 'alert-bar' row and 'current v2' column.

![image](https://github.com/WPMedia/arc-themes-blocks/assets/103296586/9f0cdddd-ce90-4ae8-9a3e-91f30e20e045)

### After

* Neither the link nor 'X' have drop shadows
* The link turns `#232B39` when hovered over with the mouse

https://github.com/WPMedia/arc-themes-blocks/assets/103296586/98618406-c4d5-43c6-b326-e862682792b2

## Dependencies or Side Effects

This PR is dependent on some changes to the `arc-themes-feature-pack`. The PR is [here](https://github.com/WPMedia/arc-themes-feature-pack/pull/418)

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1191]: https://arcpublishing.atlassian.net/browse/THEMES-1191?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
